lib/fastlane/plugin/wpmreleasetoolkit/actions/common/promo_screenshots_action.rb

```diff
@@ -238,6 +173,75 @@ def self.available_options
       def self.is_supported?(platform)
         true
       end
+
+      def self.build_entries(config, languages, output_directory, params)
```

Comment:
I like this, but AFAICT `config` isn't used directly and we could just pass the entries. 

Might not be worth fixing just now

Reply:
Nice catch! Fixed in fd8d9bde9600caa7a283eb1e46600811c3a03d50

(That refactoring might go even further I guess, but didn't want to spend too much time on it this time around as I preffered to focus on move things forward in order to put rubocop stuff behind me ASAP. Might revisit this in the future)